docs-conceptual/azps-2.0.0/migrate-az-1.0.0.md

```diff
@@ -1,26 +1,31 @@
 ---
-title: Breaking changes for Microsoft Azure PowerShell Az 1.0.0
+title: All changes from AzureRM to Azure PowerShell Az 1.0.0
 description: This migration guide contains a list of breaking changes made to Azure PowerShell in the Az version 1 release.
 author: sptramer
 ms.author: sttramer
 manager: carmonm
 ms.devlang: powershell
 ms.topic: conceptual
-ms.date: 12/14/2018
+ms.date: 05/15/2019
 ---
+# Breaking changes for Az 1.0.0
 
-# Migration Guide for Az 1.0.0
+This document provides detailed information on the changes between AzureRM 6.x and the new Az
+module, version 1.x and later. The table of contents will help guide you through a full migration
+path, including module-specific changes that may affect your scripts.
 
-This document describes the changes between the 6.x versions of AzureRM and Az version 1.0.0.
+For general advice on getting started with a migration from AzureRM to Az, see [Start migration from
+AzureRM to Az](migrate-from-azurerm-to-az.md).
 
 ## Table of Contents
```

Comment:
Style-wise, we do not do TOCs within documents. All H2 headers are automatically added to the right-hand Nav pane.

Reply:
Normally I would agree, but the RHS gutter isn't appropriate for an article like this, which needs many easily navigable subheadings with long titles. I think release notes are a special case that aren't handled particularly well by the azure-docs infrastructure and guidelines.